src/d3a/models/area/markets.py

```diff
@@ -87,14 +85,14 @@ def _delete_past_markets(self, past_markets, timeframe):
             for pm in delete_markets:
                 if ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS:
                     past_markets[pm].redis_api.stop()
-                past_markets[pm].offers = {}
-                past_markets[pm].trades = {}
-                past_markets[pm].offer_history = {}
-                past_markets[pm].notification_listeners = {}
-                past_markets[pm].bids = {}
-                past_markets[pm].bid_history = {}
-                past_markets[pm].traded_energy = {}
-                past_markets[pm].accumulated_actual_energy_agg = {}
+                del past_markets[pm].offers
+                del past_markets[pm].trades
+                del past_markets[pm].offer_history
+                del past_markets[pm].notification_listeners
+                del past_markets[pm].bids
+                del past_markets[pm].bid_history
+                del past_markets[pm].traded_energy
+                del past_markets[pm].accumulated_actual_energy_agg
```

Comment:
Really? Does it make a difference or is it "just" a cleaner way?

Reply:
Should not make a difference. I was trying to make it consistent with the IAAs actually. It also is clearer to my eyes the intent of these calls (essentially a `del` indicates that the argument does not plan to be reused, contrary to `{}` which indicates that the argument will be reused - to my eyes at least)

Reply:
Agreed! Thanks!